### PR TITLE
Fix(app.tsx outlet)

### PR DESCRIPTION
**Pull request by Pongpon00**
Create outlet of navbar and sidebar to render properly

### CODE REVIEW

File: src/App.tsx
Comment (sokungz01) [inline src/App.tsx:1]: Please resolved the conflict, then please re-assigned for review the code.

**Review by sokungz01 — CHANGES_REQUESTED** [review]
Resolved the merge conflict.

File: src/App.tsx
Comment (Pongpon00) [reply src/App.tsx:1]: done

**Review by sokungz01 — APPROVED** [review]
lgtm 👋

**Pull request merged**